core/source/util/romsizes.cpp

```diff
@@ -28,12 +28,20 @@ unsigned int FunkyBoy::romSizeInBytes(ROMSize size) {
             return 256 * 1024;
         case ROM_SIZE_512K:
             return 512 * 1024;
-        case ROM_SIZE_1024K:
+        case ROM_SIZE_1M:
             return 1024 * 1024;
-        case ROM_SIZE_2048K:
+        case ROM_SIZE_2M:
             return 2048 * 1024;
-        case ROM_SIZE_4096K:
+        case ROM_SIZE_4M:
             return 4096 * 1024;
+        case ROM_SIZE_8M:
+            return 8 * 1024 * 1024;
+        case ROM_SIZE_1_1M:
```

Comment:
Let's just disable those in-between ROM sizes for now until we can guarantee compatibility